Publication 20260601 (#186)
jjon-saxton/yaya-world: content/en/news/2026/youtube-should-do-better.org

#+TITLE: YouTube Should Do Better
#+DATE: 2026-05-30T14:31:37-07:00
#+DRAFT: false
#+DESCRIPTION:
#+TAGS[]:
#+KEYWORDS[]:
#+SLUG:

> #+SUMMARY: Quite frankly, though I am not a YouTube Partner yet, this was handled by YouTube Partner Support and they should be trained to treat everyone they contact as a partner. This is not how a partner is treated.

Reviewer: but?

```suggestion
#+SUMMARY: Quite frankly, though I am not a YouTube Partner yet, but this was handled by YouTube Partner Support, and they should be trained to treat everyone they contact as a partner. This is not how a partner is treated.
```

> Good news everyone! I have uploaded almost a third of my videos, shorts, and livestreams to my [[https://odysee.com/@yayoivt:8?r=fFewSdBCc59wLzVqnURiuU1YteYZbNae][Odysee Channel]]! Over the next several site updates I'll be switching out my YouTube Embeds with embedded videos from Odysee instead. This was never really a plan, but, [[{{% ref "news/2026/youtube-and-copyright.org" %}}][recent events]] have taught me that I need another place for my archives. I can't really host several hundre gigabites of video here and there weren't very many other options available, so Odysee it is! If Odysee were to go down for some reason am not really sure what I'd do, but, at least for now, there are still the YouTube archives.

Reviewer: hundred

```suggestion
Good news everyone! I have uploaded almost a third of my videos, shorts, and livestreams to my [[https://odysee.com/@yayoivt:8?r=fFewSdBCc59wLzVqnURiuU1YteYZbNae][Odysee Channel]]! Over the next several site updates I'll be switching out my YouTube Embeds with embedded videos from Odysee instead. This was never really a plan, but [[{{% ref "news/2026/youtube-and-copyright.org" %}}][recent events]] have taught me that I need another place for my archives. I can't really host several hundred gigabites of video here and there weren't very many other options available, so Odysee it is! If Odysee were to go down for some reason am not really sure what I'd do, but, at least for now, there are still the YouTube archives.
```

> The TL;DR of what happened was that a rescent livestream was taken down by YouTube for what they believe was a vialation of their 3rd Party Content Guidelines. I appealed stating that I had permission to use the content, but they automatically rejected my appeal. I had taken to Twitter to reach out and have been in contact with a moderatately helpful YouTube Partner Support agent who goes by Wendy. I'll go over the conversation and the next steps for a potential resolution now that you should be caught up.

Reviewer: recent/violation/we
```suggestion
The TL;DR of what happened was that a recent livestream was taken down by YouTube for what they believe was a violation of their 3rd Party Content Guidelines. We appealed stating that I had permission to use the content, but they automatically rejected my appeal. I had taken to Twitter to reach out and have been in contact with a moderatately helpful YouTube Partner Support agent who goes by Wendy. I'll go over the conversation and the next steps for a potential resolution now that you should be caught up.
```

Partners would look out for one another. Partners would help each other. Partners would share the load. YouTube is owned by Google. Google has the second largest market cap in the world. They certainly have resources they could use to support small creators. It is my opinion that they choose not to allocate those resources to us. This is why they blame us if something goes wrong and why they refuse to support us. This situation could have and /should/ have been handled by YouTube/Google. Their system may have identified and actual breach of copyright, but they failed to identify that it had and failed to verify my claim that I had permission. Instead they expect me do all the work.
#+attr_html: :align center :width 100% :title E-Mail from Wendy :alt Since your appeal was rejected, you're welcome to simply wait for your Community Guidelines warning to expire 90 days after completing the policy training. However, you may also reach out to the content owner and ask that they release the claim on your live stream so the warning can be removed earlier than that.
[[/~yayoi/images/wendy3.png]]

> I was expected to either take the training or contact the claiment for them to release the claim. This is simply not a Yayoi problem, this is a YouTube problem. Again, Google has the second largest market gap on the planet, and I'm a part-time vTuber just trying to do her best to entertain. They have plenty of resources to have acted on my behalf in this situation, and this could have and /should/ have been done as part of the appeal process. If we are really partners, this isn't unreasonable as YouTube simply has more resources.

Reviewer: you were actually expected to do both :P

```suggestion
I was expected to take the training and contact the claimant for them to release the claim. This is simply not a Yayoi problem. This is a YouTube problem. Again, Google has the second largest market gap on the planet, and I'm a part-time vTuber just trying to do her best to entertain. They have plenty of resources to have acted on my behalf in this situation, and this could have and /should/ have been done as part of the appeal process. If we are really partners, this isn't unreasonable as YouTube simply has more resources.
```

* If you value feedback, don't dismiss it
#+attr_html: :align center :width 100% :title E-Mail from Wendy :alt While I appreciate your feedback about the appeal
process, an appeal option will remain available to you when this happens.
[[/~yayoi/images/wendy1.png]]

> Finally, this language is very dismissive. You you really value and appreciate feedback, to something with it. Saying "While I appreciate your feedback about the appeal process, an appeal option will remain available to you when this happens." After I tell you how I feel it is broken and should be fixed or removed. Is simply dismissive. You could have said: "While I appreciate your feedback about the appeal

Reviewer: If you/do something

```suggestion
Finally, this language is very dismissive. If you really value and appreciate feedback, do something with it. Saying "While I appreciate your feedback about the appeal process, an appeal option will remain available to you when this happens." After I tell you how I feel it is broken and should be fixed or removed. Is simply dismissive. You could have said: "While I appreciate your feedback about the appeal
```
